### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by Dev-AkshatGupta**
Hey guys done with the landing page. Please review this thanks 
Live pre-view-https://piston-mart.netlify.app/

### CODE REVIEW

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good to me. Just your banner image is repeating. Fix that

File: css-files/root.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (partha8) [inline css-files/root.css:36]: There's a huge spacing here.

**Review by partha8 — COMMENTED** [review]
Screenshot 2022-02-06 003409

There's some space as you can see, you might want to look into it.

File: css-files/root.css (hunk 1 above)
Comment (Dev-AkshatGupta) [reply css-files/root.css:36]: Thanks to review man I will remove it

**Pull request merged**